### ISSUE TITLE
Other DB options for gorm

### ISSUE THREAD

**Issue by fnxln** [unmojang/drasl#47]
# What would be implemented
Other options for databases other than sqlite
# Why
for perfomace reasons, such as postgres or mariadb/mysql,
gorm support that natively and would be easy to implement

# References

* https://gorm.io/docs/connecting_to_the_database.html

**Comment by evan-goode** [comment]
Yes, it'd be nice to support Postgres and maybe MariaDB eventually, but it's a low priority for me at the moment. I don't have reason to believe that Postgres would be faster than SQLite except perhaps for very large instances. Skin and cape blobs get written straight to the filesystem, not the database, so the amount of data Drasl needs to write to the DB at a given time just isn't that high.

But I stand to be corrected. And there are plenty of reasons other than performance to use Postgres over SQLite. Maybe it could be a good issue for someone else to work on?
